app/celer-dump-data.cc

```diff
@@ -339,29 +339,20 @@ void print_volumes(std::vector<ImportVolume>&   volumes,
 /*!
  * Print EM parameters.
  */
-void print_em_params(ImportData::ImportEmParamsMap& em_params_map)
+void print_em_params(const ImportEmParameters& em_params)
 {
-    if (em_params_map.empty())
-    {
-        CELER_LOG(info) << "EM Parameters not available";
-        return;
-    }
-
-    CELER_LOG(info) << "Loaded " << em_params_map.size() << " EM parameters";
-
+#define PEP_STREAM_PARAM(NAME) \
+    "| " << setw(18) << #NAME << " | " << setw(7) << em_params.NAME << " |\n"
     cout << R"gfm(
 # EM parameters
 
 | EM parameter       | Value   |
 | ------------------ | ------- |
-)gfm";
-
-    for (const auto& key : em_params_map)
-    {
-        cout << "| " << setw(18) << to_cstring(key.first) << " | " << setw(7)
-             << key.second << " |\n";
-    }
-    cout << endl;
+)gfm" << std::boolalpha
+         << PEP_STREAM_PARAM(energy_loss_fluct) << PEP_STREAM_PARAM(lpm)
```

Comment:
Table looks off when printed. Not sure why, since the macro looks right at first sight...

Reply:
This is a weird one to crack... looks like a combination of std::boolalpha and std::setw causing the `" |\n"` to be right-aligned and indented...

Reply:
Ugh. https://timsong-cpp.github.io/lwg-issues/2703

Reply:
So
```
   cout << std::boolalpha << "| " << setw(18) << false << "|\n";
   cout << std::noboolalpha << "| " << setw(18) << false <<  "|\n";
```
prints
```
| false|
                | 0                 |
```

Reply:
I put in a workaround that drops those, yes.